controller/rpc/pb/controller.proto

```diff
@@ -23,6 +23,11 @@ service ControllerService {
     rpc ReplicaPrepareRebuild(ReplicaAddress) returns (ReplicaPrepareRebuildReply);
     rpc ReplicaVerifyRebuild(ReplicaAddress) returns (ControllerReplica);
 
+    rpc BackupReplicaMapCreate(BackupReplicaMap) returns (google.protobuf.Empty);
+    rpc ReplicaAddressGet(ReplicaAddressRequest) returns (ReplicaAddressResponse);
+    rpc BackupGet(google.protobuf.Empty) returns (BackupGetResponse);
```

Comment:
It's a list, not get.

Also, it's not `get backup`, it just `get backupID`.

You probably just do `BackupReplicaMapList` and return a map contains all the backup IDs and replica IPs.